### ISSUE TITLE
Enable Webcam redirect just like the audio/mic

### ISSUE THREAD

**Issue by felipemachado12** [iiordanov/remote-desktop-clients#541]
Hello
I'm new on this software, but I'm missing a webcam redirect support, just like the microphone does.

Is it supported?(maybe I'm just dumb and didn't find it on settings)

If not, can you make it?

Thanks!

**Comment by iiordanov** [comment]
I don't believe such capability is possible. The audio redirection is
something specifically supported by the SPICE protocol, but I haven't seen
anything about webcams. USB devices may be redirected, but I have not
tested with a usb webcam and I am not quite sure such high-bandwidth
applications would work well.

**Comment by felipemachado12** [comment]
Is this protocol different from native Microsoft RDP? I'm asking because the official Microsoft RDP Client can redirect (but the mouse/trackpad is awful, and your app is better in a lot of other things). I'm using it localhost, so bandwidth is not a problem in my case, of course that will be when I go remote, but in that case, I'll just not enable it.

But, if this isn't possible, well... Thanks anyway!

**Comment by iiordanov** [comment]
Ah, so you are talking about aRDP, not aSPICE. Apologies. Yes, it's totally
different from Microsoft RDP. It's fully open source software and uses
libfreerdp to talk over the RDP protocol. I will look into whether
libfreerdp does or can be made to support webcam redirection.

**Comment by felipemachado12** [comment]
Yes, it's about aRDP, sorry I didn't specify in my first comment. Thank you very much, if you be able to implement this feature this app would be THE GOAT!
